app/src/main/java/org/hackillinois/android/view/ProjectAdapter.kt

```diff
@@ -22,13 +22,19 @@ class ProjectAdapter internal constructor(private val mentorList: List<ProjectMo
         return viewHolder
     }
 
-    override fun onBindViewHolder(holder: ProjectAdapter.ViewHolder, position: Int) {
+    override fun onBindViewHolder(holder: ViewHolder, position: Int) {
         val mentor = mentorList[position]
 
+        var mentors = ""
```

Comment:
Can you put this block of code that creates the mentors string as a function in ProjectModel?

It could be something like `getMentorsString()`